app/routes/index.js

```diff
@@ -14,7 +16,36 @@ router.get('/login', async function(req, res, next){
 });
 
 router.post('/login', async function(req, res, next){
-  // TODO: fill in login logic
+  try {
+
+    const user = req.body;
+    const user_in_db = await usersModel.getByEmail(user.email);
+
+    if (user_in_db.length === 0){
+      // eventually need to make a better page and redirect them to signup.
+      return res.status(401)
+          .json({status: 'failed', message: 'Email not found, user does not exist in database'});
+    }
+
+    // add password comparison bcrypt.compareSync(user.password, user_in_db[0].password) returns boolean
+    if(!bcrypt.compareSync(user.password, user_in_db[0].password)){
```

Comment:
Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443778)
```suggestion
    if (!bcrypt.compareSync(user.password, user_in_db[0].password)){
```